big_tests/tests/graphql_SUITE.erl

```diff
@@ -199,7 +199,7 @@ listener_reply_with_validation_error(Config) ->
     Ep = ?config(schema_endpoint, Config),
     Body = #{<<"query">> => <<"query Q1 { field } query Q1 { field }">>,
              <<"operationName">> => <<"Q1">>},
-    {Status, Data} = execute(Ep, Body, undefined).
```

Comment:
I think that this test should be wanting to verify the returned value, shouldn't it? 🤔